services/venice-server/src/main/java/com/linkedin/venice/listener/ReadQuotaEnforcementHandler.java

```diff
@@ -188,30 +204,34 @@ public void channelRead0(ChannelHandlerContext ctx, RouterRequest request) {
     String storeName = request.getStoreName();
     Store store = storeRepository.getStore(storeName);
 
-    if (checkStoreNull(ctx, request, null, false, store)) {
+    if (!enforcing || checkStoreNull(ctx, request, null, false, store)) {
```

Comment:
Just double check - what does this enforcing mean? I see the setEnforcing() method is not used, which means it is always true?

Reply:
Would like to get some explanation for this as it is basically blocking me understanding the change :)

Reply:
Yes, it should always be enforced. Let me remove it. I think this is legacy code where initially the author wanted to rollout in phases where there is a mode to log/report quota but not actually reject them.